### ISSUE TITLE
Compilation fails with unthreaded perls (1.013)

### ISSUE THREAD

**Issue by eserte** [tonycoz/imager#472]
See subject:
```
cc -c   -fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -D_FORTIFY_SOURCE=2 -O2   -DVERSION=\"1.013\" -DXS_VERSION=\"1.013\" -fPIC "-I/opt/perl-5.26.2D/lib/5.26.2/x86_64-linux-ld/CORE"  -DOS_linux   Imager.c
In file included from Imager.xs:6:
Imager.xs: In function 'S_im_context_DESTROY':
/opt/perl-5.26.2D/lib/5.26.2/x86_64-linux-ld/CORE/perl.h:6477:19: error: 'my_cxt' undeclared (first use in this function); did you mean 'my_exit'?
 #  define MY_CXT  my_cxt
                   ^~~~~~
Imager.xs:91:14: note: in expansion of macro 'MY_CXT'
   if (ctx == MY_CXT.ctx)
              ^~~~~~
/opt/perl-5.26.2D/lib/5.26.2/x86_64-linux-ld/CORE/perl.h:6477:19: note: each undeclared identifier is reported only once for each function it appears in
 #  define MY_CXT  my_cxt
                   ^~~~~~
Imager.xs:91:14: note: in expansion of macro 'MY_CXT'
   if (ctx == MY_CXT.ctx)
              ^~~~~~
make: *** [Makefile:561: Imager.o] Error 1
```

**Comment by tonycoz** [comment]
Thanks, fixed in git.  I've also added a non-threaded build to CI.

I expect I'll release it later today (AU time)

**Comment by tonycoz** [comment]
Fixed in 1.014 just released
